sample_tpl/cte_scan_temp_table_insert.tpl

```diff
@@ -0,0 +1,56 @@
+// INSERT INTO temp_table SELECT colA FROM test_1 WHERE colA BETWEEN 495 AND 505
+// Select * from temp_table
+// Returns the number of tuples inserted (11)
+fun main(execCtx: *ExecutionContext) -> int32 {
+  // Init inserter
+  var col_oids: [1]uint32
+  col_oids[0] = 1 // colA
```

Comment:
Can you use the testCatalogLookup builtin? The intent is to prevent strange .tpl file breakage when test tables are edited.